src/api.lisp

```diff
@@ -22,14 +24,24 @@ All Lisp calls will get wrapped in a block named NAME, within which a HANDLER-BI
      (defmethod error-map-type ((error-map (eql ',name)))
        ',error-type)
      (defmethod error-map-success-code ((error-map (eql ',name)))
-       ,no-error)))
+       ,no-error)
+     (defmethod error-map-fatal-code ((error-map (eql ',name)))
+       ,fatal-error)))
 
 (defun c-to-lisp-name (c-name)
   (nsubstitute #\- #\_ (string-upcase c-name)))
 
 (defun lisp-to-c-name (lisp-name)
   (nsubstitute #\_ #\- (string-downcase (symbol-name lisp-name))))
 
+(defun callable-name-with-c-prefix (callable-name prefix)
+  (multiple-value-bind (c-name lisp-name)
```

Comment:
Though you can' tell by looking at the sbcl-librarian codebase, docstrings are always nice.

In this case, b/c the data-processing behavior of this function is fairly straightforward, it'd be nice to explain the role it plays. Either in a docstring or a comment.

Reply:
Docstring added